### DISCUSSION TITLE
Output files from .m4a sources suddenly quiet

### DISCUSSION THREAD

**Discussion by rockcake45** [enzo1982/freac#508] [Help & Support]
Hi All,
I just noticed that whenever I try to convert .m4a files (from iTunes) into other formats, the output files are all really quiet (about -12dB peak volume); this is a sudden, new problem.  Other types of source files give normal results.  I'm using the x86 appimage on Linux Mint 20.1, if that helps.  I've tried upgrading Freac from 1.1.6 to 1.1.7 and deleting the .config file with no success.   I really like Freac & would like to keep using it, but this is a significant part of my use of it.  Thanks in advance.

**Comment by Lee-Carre** [comment]
> convert .m4a files (from iTunes) into other formats, the output files are all really quiet (about -12dB peak volume)

Hmm, that is an odd problem.

Which format/codec are you outputting to, if that's the only one which exhibits this problem/behaviour/characteristic?

While not a solution to a potential bug, as a work-around you could use the volume-changer filter/plugin (I'm recalling from memory, since my instance of Fre:AC is busy encoding, currently).

> about -12dB peak volume

What's the typical loudness of other formats/codecs, for comparison (since the general results of output depend on the nature & quality of the input)?

> Other types of source files give normal results.

Ah, actually, now that I've re-read more carefully; it could well be that the problem is something with the source files.

~~What happens when you use a different source, but the same encoding configuration as for your M4As from iTu[d]es?~~ You already answered that; I must be tired.

Have you performed an acoustic analysis on those M4As, before any transcoding? Maybe those sources are relatively quiet, but iTu[d]es is doing some kind of compensation (ReplayGain? Dynamic range compression?).

> I'm using the x86 appimage on Linux Mint 20.1

Presuming that your host is 64-bit, wouldn't a 64-bit build be preferable? I'm not familiar with Mint, though.

**Comment by slarti76** [comment]
I noticed the same problem. Not with all files, e.g. I just bought 3 albums of a band, the newest from last year and 2 older. The older ones converted 1:1, the newer was much quieter and I had to activate that volume adjust post-proc. In this case, +14dB was good value.
So I really think Bapple did sth in newer files that throws fre:ac off...
